src/CMakeLists.txt

```diff
@@ -55,6 +55,32 @@ set(SOURCE_FILES
 add_library(EIPScanner SHARED ${SOURCE_FILES})
 add_library(EIPScannerS STATIC ${SOURCE_FILES})
 
+message(${PROJECT_SOURCE_DIR}/src)
```

Comment:
This should not be included in the pull request. This looks like debug logging

Reply:
I link it to my project and I had some troubles with includes. How should I try to modify CMakeLists.txt?